public/json/search_in_dict.json

```
@@ -0,0 +1,27 @@
+{
+    "title": "Search in Dictionary.app",
+    "author": "Ramiro Garcia (https://github.com/ranemirusG)",
+    "rules": [
+        {
+            "description": "Use LEFT OPTION + s to search copied word in Dictionary.app",
+            "manipulators": [
+                {
+                    "type": "basic",
+                    "from": {
+                        "key_code": "s",
+                        "modifiers": {
+                            "mandatory": [
+                                "left_option"
+                            ]
+                        }
+                    },
+                    "to": [
+                        {
+                            "shell_command": "open dict://`pbpaste` " 
```

It is better to enclose it in double quotes for the result of pbpaste contains spaces, for example "hello world".

```suggestion
                            "shell_command": "open dict://`pbpaste` " 
                            "shell_command": "open \"dict://`pbpaste`\""                             
```